check_ruby.sh

```diff
@@ -44,7 +44,7 @@ pushd "${PROJECT_REPO}" > /dev/null || exit 1
 
     if [ -f "${script_file}" ] &&
        ( [[ ${script_file} == *.rb ]] ||
-      grep '^#!/bin/.*ruby' "${script_file}" ); then
+         (head -1 "$script_file" | grep -E '^#!(/usr)?/bin/.*ruby'); then
```

Comment:
(lint) Unexpected keyword/token. Fix any mentioned problems and try again. [SC1072]